### PR TITLE
Add test cases to verify be-BY(Belarusian-Belarus)

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Added test cases to verify be-BY (Belarusian-Belarus). Few tests have existed already.
* timezone: Europe/Minsk https://www.zeitverschiebung.net/en/timezone/europe--minsk

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/test/date/testdatefmt_be_BY.js [hunk 1]
```diff
@@ -0,0 +1,1788 @@
+/*
+ * testdatefmt_be_BY.js - test the date formatter object in Georgian
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:2]: Georgian -> Belarus

File: js/test/date/testdatefmt_be_BY.js [hunk 2]
```diff
@@ -0,0 +1,1788 @@
...
+    },
+
+    testDateFmtFullTimeComponentsHMZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            length: "full", 
+            time: "hmz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1255]: The time zone used in Belarus is "Europe/Minsk", which is similar to "Europe/Moscow" but different.

File: js/test/date/testdatefmt_be_BY.js [hunk 3]
```diff
@@ -0,0 +1,1788 @@
...
+        test.done();
+    },
+
+    testDateFmtShortTimeComponentsHMZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            time: "hmz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1006]: The time zone used in Belarus is "Europe/Minsk", which is similar to "Europe/Moscow" but different.

File: js/test/date/testdatefmt_be_BY.js [hunk 4]
```diff
@@ -0,0 +1,1788 @@
...
+        test.done();
+    },
+
+    testDateFmtShortTimeComponentsHMAZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            time: "hmaz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1030]: Europe/Minsk

File: js/test/date/testdatefmt_be_BY.js [hunk 5]
```diff
@@ -0,0 +1,1788 @@
...
+        test.done();
+    },
+
+    testDateFmtShortTimeComponentsHMSZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            time: "hmsz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1073]: Europe/Minsk

File: js/test/date/testdatefmt_be_BY.js [hunk 6]
```diff
@@ -0,0 +1,1788 @@
...
+        test.done();
+    },
+
+    testDateFmtShortTimeComponentsHMSAZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            time: "hmsaz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1097]: Europe/Minsk

File: js/test/date/testdatefmt_be_BY.js [hunk 7]
```diff
@@ -0,0 +1,1788 @@
...
+    },
+
+    testDateFmtFullTimeComponentsHMAZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            length: "full", 
+            time: "hmaz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1280]: Europe/Minsk

File: js/test/date/testdatefmt_be_BY.js [hunk 8]
```diff
@@ -0,0 +1,1788 @@
...
+    },
+
+    testDateFmtFullTimeComponentsHMSZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            length: "full", 
+            time: "hmsz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1324]: Europe/Minsk

File: js/test/date/testdatefmt_be_BY.js [hunk 9]
```diff
@@ -0,0 +1,1788 @@
...
+    },
+
+    testDateFmtFullTimeComponentsHMSAZ_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            length: "full", 
+            time: "hmsaz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1349]: Europe/Minsk

File: js/test/date/testdatefmt_be_BY.js [hunk 10]
```diff
@@ -0,0 +1,1788 @@
...
+    },
+
+    testDateFmtWithTimeZoneAndNoDST_be_BY: function(test) {
+        test.expect(2);
+        var fmt = new DateFmt({
+            locale: "be-BY", 
+            type: "time", 
+            length: "full", 
+            time: "hmsz", 
+            timezone: "Europe/Moscow"
```
Comment (ehoogerbeets) [inline js/test/date/testdatefmt_be_BY.js:1374]: Europe/Minsk

File: js/test/daterange/testdatefmtrange_be_BY.js [hunk 11]
```diff
@@ -0,0 +1,692 @@
+/*
+ * testdatefmtrange_be_BY.js - test the date range formatter object Georgian/Georgia
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_be_BY.js:2]: Belarussian/Belarus

File: js/test/root/teststrings.js [hunk 12]
```diff
@@ -3263,6 +3263,46 @@ module.exports.teststrings = {
...
+    testStringFormatChoiceCharClasses27: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("be-BY");
+
+        test.ok(str !== null);
+
+        test.equal(str.formatChoice(3), "The items are few");
+        test.done();
+    },
```
Comment (ehoogerbeets) [inline js/test/root/teststrings.js:3295]: Need a test case for the plurals between 5-9, which should be the "many" case

File: js/test/strings-ext/testlistfmt_be_BY.js [hunk 13]
```diff
@@ -0,0 +1,253 @@
...
+    testListFmtUnitStylekaGENumberFormatFiveFull: function(test) {
+        var fmt = new ListFmt({
+            locale: "be-BY",
+            style: "unit",
+            length: "full"
+        });
+
+        test.expect(2);
+        test.ok(fmt !== null);
+        test.equal(fmt.format(["адзін", "два", "тры", "чатыры", "ხუთი"]), "адзін два тры чатыры ხუთი");
```
Comment (ehoogerbeets) [inline js/test/strings-ext/testlistfmt_be_BY.js:195]: ხუთი -> not cyrillic characters. Where did that come from?

Comment (gouniLee) [reply js/test/strings-ext/testlistfmt_be_BY.js]: Probably I missed it when I copied texts. I updated it from the google translate.

File: js/test/root/teststrings.js (hunk 12 above)
Comment (gouniLee) [reply js/test/root/teststrings.js:3295]: I added it

File: js/test/daterange/testdatefmtrange_be_BY.js (hunk 11 above)
Comment (gouniLee) [reply js/test/daterange/testdatefmtrange_be_BY.js]: Updated it

File: js/test/date/testdatefmt_be_BY.js (hunk 3 above)
Comment (gouniLee) [reply js/test/date/testdatefmt_be_BY.js]: oh, I updated all with the correct timezone.

**Comment by ehoogerbeets** [comment]
Also, please update the year in the copyright notice for each file you changed to include 2020.

File: js/test/units/testunitfmt_be_BY.js [hunk 14]
```diff
@@ -0,0 +1,330 @@
...
+            locale: "be-BY",
+            autoConvert: true,
+            length: "short"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "3,21868 Mm");
+        test.done();
+    },
+    testUnitFormatLength2_be_BY: function(test) {
+        test.expect(1);
```
Comment (ehoogerbeets) [inline js/test/units/testunitfmt_be_BY.js:207]: Test a different measure that is translated to Belarussian...

Comment (gouniLee) [reply js/test/units/testunitfmt_be_BY.js:207]: I updated it

**Pull request merged**